src/providers/authentication/__tests__/authentication.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { TestBed } from '@angular/core/testing';
+import { InAppBrowser } from '@ionic-native/in-app-browser';
+import { MSAdal } from '@ionic-native/ms-adal';
+
+import { MSAdalMock } from '../__mocks__/ms-adal.mock';
+import { AuthenticationProvider } from '../authentication';
+import { AppConfigProvider } from '../../app-config/app-config';
+import { AppConfigProviderMock } from '../../app-config/__mocks__/app-config.mock';
+import { InAppBrowserMock } from '../__mocks__/in-app-browser.mock';
+
+describe('Authentication', () => {
+  let authenticationProvider: AuthenticationProvider;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
```

Comment:
Do you need TestBed to test this class? Or is it just easier than managing the mock lifecycle by hand?